src/components/Home/HomeNewsItem/index.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { View, Text, Linking } from 'react-native';
+import { TouchableOpacity } from 'react-native-gesture-handler';
+
+import SvgContainer from '../../SvgContainer';
+import styles from './styles';
+
+const badgeMetadata = {
+  height: '45',
+  width: '45',
+};
+
+interface HomeNewsItemProps {
+  newsTitle: string;
+  newsText: string;
+  link: string;
+}
+
+const openLink = (url: string) => {
+  Linking.canOpenURL(url).then(supported => {
+    if (supported) {
+      Linking.openURL(url);
+    } else {
+      console.log(`Don't know how to open URI: ${url}`);
```

Comment:
maybe an `alert` here over a console.log? just thinking from a user perspective, since this behavior isn't well defined on the figma. Thoughts?

Reply:
Oops yeah meant to change this. Alert sounds good!